packages/bpk-component-popover/src/BpkPopoverPortal.js

```diff
@@ -42,21 +44,64 @@ const onOpen = (popoverElement, targetElement) => {
   }
 };
 
-class BpkPopoverPortal extends Component {
+export type Props = {
+  ...$Exact<PopoverProps>,
+  target: Function | Node,
+  isOpen: boolean,
+  onClose: Function,
+  tetherOptions: Object,
```

Comment:
I feel like we should list the exact possible options here, but since we might replace tether, let's defer that

Reply:
That was my thinking :P